src/utils/common.ts

```diff
@@ -176,19 +176,19 @@ export function postJson(url: string, options?: IRequestInitBodyParsed) {
   });
 }
 
-export function secondsToRelativeTime(seconds: number) {
+export function secondsToRelativeTime(seconds: number, shortForm?: boolean) {
   const { t } = useI18n();
   const secondsPerMinute = 60;
   const secondsPerHour = secondsPerMinute * 60;
   const secondsPerDay = secondsPerHour * 24;
   if (seconds < secondsPerMinute) {
-    return t('common.seconds', Math.round(seconds));
+    return t(shortForm ? 'common.secondsShort' : 'common.seconds', Math.round(seconds));
```

Comment:
Probably the transpiler will complain about trying to dynamically establish the translation key. So I suggest to avoid it by doing this:
```suggestion
    return (shortForm)
      ? t('common.secondsShort', Math.round(seconds))
      : t('common.seconds', Math.round(seconds));
```

Applies to next 2 returns.